src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -2360,11 +2380,11 @@ private void PopulateInterfaceImplTableRows()
             foreach (ITypeDefinition typeDef in this.GetTypeDefs())
             {
                 var typeDefHandle = GetTypeDefinitionHandle(typeDef);
-                foreach (ITypeReference interfaceRef in typeDef.Interfaces(Context))
```

Comment:
Perhaps we can also fix https://github.com/dotnet/roslyn/issues/3905 now that we are changing this?